### PR TITLE
refactor: standalone

### PR DESCRIPTION
**Pull request by jlopezcur**
Moving this proyect to standalone structure.

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -1,7 +1,3 @@
 {
-  "endOfLine": "lf",
-  "semi": true,
-  "singleQuote": true,
-  "tabWidth": 2,
-  "trailingComma": "es5"
+  "singleQuote": true
```
Comment (trigoporres) [inline .prettierrc:2]: Why did we remove these rules?, I have added them to the website

File: data/course.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
+{
+  "title": "Your course title",
+  "description": "Your course description",
+  "version": "0.1.0",
```
Comment (trigoporres) [inline data/course.json:4]: we added a version to the courses, which gives us this parameter?

File: data/course.json [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  "authors": ["@dislearning"],
+  "start": "presentation",
+  "toc": [
+    {
+      "type": "text",
+      "slug": "presentation",
+      "title": "Presentation"
+    },
+    {
+      "type": "section",
```
Comment (trigoporres) [inline data/course.json:14]: I like to add sections

File: .prettierrc (hunk 1 above)
Comment (jlopezcur) [reply .prettierrc:2]: These are the default values as mentioned on: https://github.com/dislearning/website/pull/1#discussion_r512072501

File: data/course.json (hunk 2 above)
Comment (jlopezcur) [reply data/course.json:4]: The capability to versioning the content based on this param (in a "on fly" mode we wouldn't have a package.json file where we are used to).

File: data/course.json (hunk 3 above)
Comment (jlopezcur) [reply data/course.json:14]: This is only a test, we need to define a good way to represent the structure of the course, so this is only for have a different element type.

**Pull request merged**